games-board/netmaumau/netmaumau-0.15.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 2015 Julian Ospald <hasufell@posteo.de>, Heiko Schaefer <heiko@rangun.de>
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit qmake-utils eutils vcs-snapshot
+
+MY_P=nmm-qt-client${PV}
+
+DESCRIPTION="Client for games-server/netmaumau, the popular card game Mau Mau"
+HOMEPAGE="http://sourceforge.net/projects/netmaumau"
+SRC_URI="https://github.com/velnias75/NetMauMau-Qt-Client/archive/V${PV}.tar.gz -> ${P}-client.tar.gz"
+
+LICENSE="LGPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="no-espeak"
```

Comment:
Once we had the discussion about a plus (+) IUSE flag...

As you know I hate to make new or edit PR's. So please modify as you like as long as to use espeak is the default behaviour.